folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -194,6 +195,7 @@ private DefaultRawMemcacheClient(
       int maxSetLength) {
     this.address = address;
     this.executor = executor;
+    this.isDirectExecutor = executor == null || Utils.isDirectExecutor(executor);
```

Comment:
Nitpicking here, but given isDirectExecutor really meaning "is a direct executor or null", how about rather naming it something like "wrapWithExecutor"? (I'm terrible at naming so I'm sure an even better name would be available)

Reply:
Sure, that's a good idea